### PR TITLE
Refactoring Awesome Books using ES6

### PR DESCRIPTION
**Pull request by tasnimchy96**
In this branch i did:

- Used ES6 modules to write modular JavaScript.
- Refactored your functions to arrow functions.
- Displayed the current date below the navigation bar using Luxon Library.
- Divided your code into modules.

### CODE REVIEW

**Review by rustam-tolipov — COMMENTED** [review]
Hi @tasnimchy96, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that did not use the correct Gitflow. In this milestone, students were encouraged to use [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow), instead of [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md). Kindly fix the issue and resubmit your code again. Thank you!

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by tasnimchy96** [comment]
@Rustamxon7 Can you please give me a hint how should i fix it?

**Comment by rustam-tolipov** [comment]
This [link](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) explains how to create Gitflow. You can look through and learn about it.

File: README.md [hunk 1]
```diff
@@ -1 +1,115 @@
...
+
+If you like this project feel free to share with your friends.
+
+## :pray: Acknowledgments
+
+I would like to thank Microverse for this amazing exercise and also like to thank for the kind collaboration of Samreen Mujtaba.
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (Tufahel) [inline README.md:115]: image

- [x] `LICENSE.md` link is not working. Kindly add it correctly. Every link you have added in the readme file should work as per the requirement of a professional readme.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @tasnimchy96 :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Professional pull request
✔️ Correct GitHub-flow
✔️ Linter checks are passing

## Required Changes ♻️

***Please kindly check the comments below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @tasnimchy96    😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Descriptive pr👍 
- linters are passing 👍 






## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**